NuRadioReco/modules/channelSineSubtraction.py

```diff
@@ -0,0 +1,249 @@
+
+import numpy as np
+import sys
+import logging
+# Add the directory where your library is located
+#sys.path.append('/Users/nozdrina.1/Documents/phys/software/NuRadioMC')
+
+import NuRadioReco.utilities as ut
+import matplotlib.pyplot as plt
+from scipy.optimize import curve_fit
+from NuRadioReco.utilities import units, fft
+
+"""
+Contains module to filter continuous wave out of the signal using sine subtraction on peaks in frequency spectrum
+"""
+
+
+def guess_amplitude(wf: np.ndarray, target_freq: float, sampling_rate: float = 3.2):
+    """
+        Estimate the amplitude of a specific harmonic in the waveform.
+
+        Args:
+            wf (np.ndarray): Input waveform (1D array).
+            target_freq (float): Target frequency for which to estimate amplitude.
+            sampling_rate (float): Sampling rate of the waveform (GHz).
+
+        Returns:
+            float: Estimated amplitude of the target frequency.
+        """
```

Comment:
Can we correct the indentation here? Aline the lines 20-29 with 19. Also in the rest of NuRadio we use this kind of syntax for function doc-strings:
```
 Paramters
 ----------
 wf: np.ndarray
     Input waveform (1D array).
target_freq:  float
     Target frequency for which to estimate amplitude.
sampling_rate: float
     Sampling rate of the waveform (GHz).

Returns
--------
ampl: float
     Estimated amplitude of the target frequency.
```
Do you think you could change this?